bundles/TinymceBundle/public/js/editor.js

```diff
@@ -23,6 +23,8 @@ pimcore.bundle.tinymce.editor = Class.create({
     },
 
     initializeWysiwyg: function (e) {
+        console.warn('TinMCE is deprecated, please use quill instead');
```

Comment:
```suggestion
        console.info('TinMCE is deprecated, please use Quill (pimcore/quill-bundle) instead');
```

Reply:
I'm wondering whether we should put this info log just at the end of `editor.js` so that it's just logged once when initializing the UI and not on every editor initialization, resulting in flooding the console. WDYT?